src/main/java/pl/onsight/wypozyczalnia/DateFilter.java

```diff
@@ -1,7 +1,10 @@
 package pl.onsight.wypozyczalnia;
```

Comment:
trzeba zmienić package DateFilter'a do modelu. juz to zrobiłem nawet ale tego nie pushnełem i miałem praktycznie cały validator do tego stringa... i cholera wie gdzie mi to znikneło. przez jakiegoś pulla czy coś.

Reply:
tak sie nazywa ta klasa i nie wiem czemu jest w tym samym miejscu co application.class :package:  :P

Reply:
no bo w sumie mi nigdzie ona nie pasuje, więc tam sobie siedzi :P